cli/medperf/utils.py

```diff
@@ -22,9 +23,42 @@
 from medperf.exceptions import InvalidEntityError
 
 
+def set_custom_config(args: dict):
+    """Function to set parameters defined by the user
+
+    Args:
+        args (dict): custom config params
+    """
+    for param in args:
+        val = args[param]
+        setattr(config, param, val)
+
+
+def load_config(profile: str) -> dict:
+    """Loads the configuration parameters associated to a profile
+
+    Args:
+        profile (str): profile name
+
+    Returns:
+        dict: configuration parameters
+    """
+    config_p = configparser.ConfigParser()
+    config_file = os.path.join(config.storage, config.config_path)
+    config_p.read(config_file)
+    # Set current profile
+    if profile == "active":
+        # Special case. Get the profile that has been assigned as active
+        profile = config_p[profile]["profile"]
+    config.profile = profile
+    return config_p[profile]
+
+
 def storage_path(subpath: str):
     """Helper function that converts a path to storage-related path"""
-    return os.path.join(config.storage, subpath)
+    server_path = config.server.split('//')[1]
+    server_path = re.sub(r"[.:]", "_", server_path)
+    return os.path.join(config.storage, server_path, subpath)
```

Comment:
1) We need to keep in mind then that we **should** use `storage_path` **and not** `config.storage` for everything in the code, except credentials and config. I double-checked, and we are good.

2) Just wondering: does it need an `os.makedirs(exist_ok=True)` for the first time the server path is created?

Reply:
1. That's true. So far, we only keep track of two things in the global storage path (config and credentials). If we start making more use of that folder we should probably create another helper function for that (probably `global_storage_path`?) For now I don't think we need it

2. No need, this is covered by the `init_storage` function, which is always executed before anything else